### DISCUSSION TITLE
Is there a shortcut for horizontal/vertical split?

### DISCUSSION THREAD

**Discussion by anta40** [wezterm/wezterm#5248] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

On iTerm, you can easily do this. Right click on terminal area and a context menu will appear.
WezTerm currently don't have this, so you have to access the menu bar.

I tried editing the config like this:
```
config.keys = {
    {
        key = '"',
        mods = 'CTRL|SHIFT|ALT',
        action = wezterm.action.SplitVertical { domain = 'CurrentPaneDomain'},
    },
    {
        key = '%',
        mods = 'CTRL|SHIFT|ALT',
        action = wezterm.action.SplitHorizontal { domain = 'CurrentPaneDomain'},
    }
}

```
And when pressing Ctrl+" or Ctrl+%, nothing happened. Perhaps my config is incorrect.

**Comment by pvater** [comment, marked as answer]
'CTRL|SHIFT|ALT' means you have to hold all 3 modifiers simultaneously, e.g. ctrl-shift-ald-% splits horizontally.

**Reply by anta40** [reply]
Aaahhh, so I pressed the wrong shortcut key. OK got it. It works now :)
